.github/workflows/release.yaml

```diff
@@ -25,10 +25,17 @@ jobs:
     permissions:
       id-token: write
     steps:
-      - uses: actions/download-artifact@v3
+      - name: Get sdist
+        uses: actions/download-artifact@v4
         with:
-          name: artifact
+          name: Packages
```

Comment:
The artifact name `Packages` comes from https://github.com/hynek/build-and-inspect-python-package/blob/20db83a776feefaf52e2b2ba11475569a64f9bd2/action.yml#L60-L64